utils/test/testingUtils.ts

```diff
@@ -72,6 +72,10 @@ export async function getLastBlockTimestamp(): Promise<BigNumber> {
   return BigNumber.from((await provider.getBlock("latest")).timestamp);
 }
 
+export async function getLastBlockTransaction(): Promise<any> {
```

Comment:
So that we can check event emissions from `DelegatedManager.constructor`. Useful for grabbing events from contract deployments in hardhat.